Dockerfile

```
@@ -79,6 +79,7 @@ RUN set -ex \
   && apk add --no-cache \
     shadow \
     su-exec \
+    gcc build-base \
```

Let's put these build-time deps. in a virtual package so its easier to add/clean-up in the future. 
```suggestion
  && apk add --no-cache --virtual .build-deps \
    gcc \
    build-base \
```